### PR TITLE
Updated Docker 4 Drupal to 5.3.3

### PR DESCRIPTION
**Pull request by kolibrid**

### CODE REVIEW

**Comment by esolitos** [comment]
The `.travis.yml` file should not be updated, it's now running the wrong tests. :)

File: docker-compose.yml [hunk 1]
```diff
@@ -2,77 +2,87 @@ version: "2"
...
+#      PHP_XDEBUG: 1
+#      PHP_XDEBUG_DEFAULT_ENABLE: 1
+#      PHP_XDEBUG_REMOTE_CONNECT_BACK: 0
+#      PHP_IDE_CONFIG: serverName=my-ide
+#      PHP_XDEBUG_REMOTE_HOST: host.docker.internal # Docker 18.03+ & Linux/Mac/Win
+#      PHP_XDEBUG_REMOTE_HOST: 172.17.0.1 # Linux, Docker < 18.03
+#      PHP_XDEBUG_REMOTE_HOST: 10.254.254.254 # macOS, Docker < 18.03
+#      PHP_XDEBUG_REMOTE_HOST: 10.0.75.1 # Windows, Docker < 18.03
     volumes:
-      - ./:/var/www/html:cached
```
Comment (esolitos) [inline docker-compose.yml]: @kolibrid As we all use MacOS the default options should use the `:cached` option, which greatly improves performances.

File: docker-compose.yml [hunk 2]
```diff
@@ -2,77 +2,87 @@ version: "2"
...
-      MYSQL_PASSWORD: drupal
-      # Local developement tweaks
-      NGINX_ERROR_LOG_LEVEL: debug
-      NGINX_STATIC_CONTENT_EXPIRES: '-1' # Force always reload static content
-      NGINX_STATIC_CONTENT_OPEN_FILE_CACHE: 'off' # nginx should not cache static content
+      NGINX_VHOST_PRESET: $NGINX_VHOST_PRESET
+#      NGINX_DRUPAL_FILE_PROXY_URL: http://example.com
     volumes:
-      - ./:/var/www/html:cached
+      - ./:/var/www/html
```
Comment (esolitos) [inline docker-compose.yml]: Same as above, `:cached` option should be set,

File: docker-sync.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+version: "2"
```
Comment (esolitos) [inline docker-sync.yml]: Docker-sync is not required anymore as Docker on MacOS has improved performances just by using the `:cached` option on mounted volumes.
Meaning: this file is not needed.

**Review by esolitos — CHANGES_REQUESTED** [review]
Good work, but some small changes required before the merge. :)

**Comment by kolibrid** [comment]
Good that you remembered! I will fix it now.

**Review by esolitos — APPROVED** [review]

**Pull request merged**